crates/eips/src/eip4844/mod.rs

```diff
@@ -9,6 +9,10 @@ pub mod env_settings;
 #[cfg(feature = "kzg")]
 pub mod trusted_setup_points;
 
+/// Module houses the builder and utils for the [EIP-4844 Blob Transaction](https://eips.ethereum.org/EIPS/eip-4844#blob-transaction)
```

Comment:
repo convention is to say the contents and let "Module houses the" be implied by context. 

`Builder and utils for the [EIP-4844 Blob Transaction](https://eips.ethereum.org/EIPS/eip-4844#blob-transaction)`